### ISSUE TITLE
latest nightly fails to start

### ISSUE THREAD

**Issue by gnunicorn** [acterglobal/a3#2014]
Also the one from yesterday. On Android it just comes up black screen, nothing happening. The one from before was fine (though I dunno the exact time I last updated). Also can't rag shake or anything.

**Comment by gnunicorn** [comment]
the lib isn't found.... :(


Lib not found!?!

```
07-31 20:00:03.646 10818 10850 I flutter : DynamicLibrary.open by lib name failed: Invalid argument(s): Failed to load dynamic library 'libacter.so': dlopen failed: cannot locate symbol "__extenddftf2" referenced by "/data/app/~~YmPg1z7IL-cq_3Um3yo2Pw==/global.acter.a3-9doZ2V2eD-hdqKiYlPhTRg==/split_config.arm64_v8a.apk!/lib/arm64-v8a/libacter.so"...
07-31 20:00:03.647 10818 10850 I flutter : DynamicLibrary.open from /data/app failed: Invalid argument(s): Failed to load dynamic library '/data/app/~~YmPg1z7IL-cq_3Um3yo2Pw==/global.acter.a3-9doZ2V2eD-hdqKiYlPhTRg==/lib/arm64/libacter.so': dlopen failed: library "/data/app/~~YmPg1z7IL-cq_3Um3yo2Pw==/global.acter.a3-9doZ2V2eD-hdqKiYlPhTRg==/lib/arm64/libacter.so" not found
07-31 20:00:03.647 10818 10850 I flutter : DynamicLibrary.open from /data/data failed: Invalid argument(s): Failed to load dynamic library '/data/data/global.acter.a3/libacter.so': dlopen failed: library "/data/data/global.acter.a3/libacter.so" not found
07-31 20:00:03.647 10818 10850 E flutter : [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Failed to load dynamic library '/data/data/global.acter.a3/libacter.so': dlopen failed: library "/data/data/global.acter.a3/libacter.so" not found
07-31 20:00:03.647 10818 10850 E flutter : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
```

**Comment by gnunicorn** [comment]
fixed with 70433fc23c21f552c4562fe7c2f8358123823dc0 .
